### ISSUE TITLE
`escaping \# comment markers is not portable` warning

### ISSUE THREAD

**Issue by chenrui333** [ddclient/ddclient#745]
👋 while regression building 3.11.2 release, seeing some build failure as below:

```
automake: warnings are treated as errors
Makefile.am:20: warning: escaping \# comment markers is not portable
autoreconf: error: automake failed with exit status: 1
ERROR: 'autoreconf -fviW all' failed
```

full build log, https://github.com/Homebrew/homebrew-core/actions/runs/10821142464/job/30022616695

**Comment by ripdog** [comment]
This is breaking the build on the AUR package as well.

**Comment by dimpase** [comment]
and the Gentoo package

**Comment by rhansen** [comment]
Fixed by commit a12398c315b9b909e57e87acf9fd3a15a0b3e213

**Comment by barracuda156** [comment]
Also affects macOS.
